sites/partners/src/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -163,13 +163,22 @@ const BuildingDetails = ({
             validation={{ required: true }}
           />
         </GridCell>
-        <Field
-          label={t("t.neighborhood")}
-          name={"neighborhood"}
-          id={"neighborhood"}
-          placeholder={t("t.neighborhood")}
-          register={register}
-        />
+        <ViewItem label={t("t.neighborhood")} className={"mb-0"}>
+          <Select
+            id={`neighborhood`}
+            name={`neighborhood`}
+            label={t("t.neighborhood")}
+            labelClassName="sr-only"
+            register={register}
+            controlClassName="control"
+            options={neighborhoodKeys}
```

Comment:
If you create a new listing, will the neighborhood be set to the JSON value, or the JSON key? The keys don't have spaces in them, but it looks like we're expecting the neighborhoods stored in the db to have spaces (i.e. human readable).

Reply:
Looks like the JSON value will be saved